frontend/src/container/NewDashboard/DashboardSettings/Variables/index.tsx

```diff
@@ -353,6 +351,10 @@ function VariablesSetting(): JSX.Element {
 							flexDirection: 'row',
 							justifyContent: 'flex-end',
 							padding: '0.5rem 0',
+							position: 'absolute',
+							top: '-56px',
```

Comment:
NIT: -ve pixel, see if this can be avoided

Reply:
will check this in the follow-up